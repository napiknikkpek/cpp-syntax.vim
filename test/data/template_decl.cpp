template <typename A, int M, A N>
class A;
template <template <typename, std::size_t, int> class Key, typename T>
class A;
template <typename... Xs, A... I, int... N>
class A;
template <>
struct A<int, N, index<i...>, index<int...>> {};
template <>
A foo()
template <>
A::B foo()
template <>
A<B> foo()
