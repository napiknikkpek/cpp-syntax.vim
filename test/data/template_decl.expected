aaaaaaaa..aaaaaaaa.t..zzz.x..t.x.
aaaaa.t.
aaaaaaaa..aaaaaaaa..aaaaaaaa..ttt..tttttt..zzz..aaaaa.ttt..aaaaaaaa.t.
aaaaa.t.
aaaaaaaa..aaaaaaaa....tt..t....t..zzz....t.
aaaaa.t.
aaaaaaaa...
aaaaaa.t.zzz..t..ttttt.t......ttttt.zzz.........
aaaaaaaa...
t.fff..
aaaaaaaa...
t..t.fff..
aaaaaaaa...
t.t..fff..
